tests/periph_i2c/tests/04__periph_i2c_flags.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation       Tests to verify correct error codes are given.
```

Comment:
Is this correct? should state something about flags, or not?

Reply:
copy paste error, I will handle it